libafl/src/executors/inprocess.rs

```diff
@@ -827,8 +827,6 @@ mod unix_signal_handler {
             ExitKind::Timeout,
         );
 
-        event_mgr.await_restart_safe();
```

Comment:
this is not needed?

Reply:
I swear if this was the cause of my issues...

Reply:
what were the issues?
cc @domenukk

Reply:
`run_observers_and_save_state` does it, so this is fine